docs/library/axi_ad485x/index.rst

```diff
@@ -0,0 +1,391 @@
+    { "name": "data 5",  "wave": "z..........................4.........","data":["ch 5"]},
+    { "name": "data 6",  "wave": "z..........................4.........","data":["ch 6"]},
+    { "name": "data 7",  "wave": "z..........................4.........","data":["ch 7"]},
+  ],
+  foot: {text: ['tspan', 'all lanesi(8) all channels(8) active']}}
+
+Register Map
+--------------------------------------------------------------------------------
+
+The register map of the core contains instances of several generic register maps
+like ADC common, ADC channel, up_delay_ctrl. The following table presents the
+base addresses of each instance, after that can be found the detailed
+description of each generic register map. The absolute address of a register
+should be calculated by adding the instance base address to the registers
+relative address.
+
+.. list-table:: Register Map base addresses for axi_ad485x
+   :header-rows: 1
+
+   * - DWORD
+     - BYTE
+     - Name
+     - Description
+   * - 0x0000
+     - 0x0000
+     - BASE
+     - See the `Base <#hdl-regmap-COMMON>`__ table for more details.
+   * - 0x0000
+     - 0x0000
+     - RX COMMON
+     - See the `ADC Common <#hdl-regmap-ADC_COMMON>`__ table for more details.
+   * - 0x0000
+     - 0x0000
+     - RX CHANNELS
+     - See the `ADC Channel <#hdl-regmap-ADC_CHANNEL>`__ table for more details.
+   * - 0x0000
+     - 0x0800
+     - IO_DELAY_CNTRL
+     - See the `I/O Delay Control <#hdl-regmap-DELAY_CNTRL>`__ table for more details.
+
+.. hdl-regmap::
+   :name: COMMON
+   :no-type-info:
+
+.. hdl-regmap::
+   :name: ADC_COMMON
+   :no-type-info:
+
+.. hdl-regmap::
+   :name: ADC_CHANNEL
+   :no-type-info:
+
+.. hdl-regmap::
+   :name: IO_DELAY_CNTRL
+   :no-type-info:
+
+Design Guidelines
+--------------------------------------------------------------------------------
+
+The IP was developed part of the
+:dokuwiki:`AD485x Native FMC Card Reference Design <resources/fpga/xilinx/fmc/ad485x>`.
+
+The control of the AD485x chip is done through a SPI interface, which is needed
+at system level.
+
+The *ADC interface signals* must be connected directly to the top file of the
+design, as IO primitives are part of the IP.
+
+The example design uses a DMA to move the data from the output of the IP to
+memory.
+
+If the data needs to be processed in HDL before moved to the memory, it can be
+done at the output of the IP (at system level) or inside of the adc channel
+module (at IP level).
+
+The example design uses a processor to program all the registers. If no
+processor is available in your system, you can create your own IP starting from
+the interface module.
+
+Software Guidelines
+--------------------------------------------------------------------------------
+
+The software for this IP can be found as part of the AD485x Native FMC Card
+Reference Design at: :git-no-OS:`projects/ad485x` (TBD)
+Linux is supported also using :git-linux:`/`.
```

Comment:
I noticed the TBD.
There is no project for no-OS, but there is a driver.
Will the project be added soon?
Maybe add a link to the driver, since it exists already?

And for Linux, maybe we should just remove it, not to create confusion.

Reply:
Removed no-OS.
Linux is not yet in it's final form. I think.